lib/graphql/language/lexer.rl

```diff
@@ -198,10 +198,10 @@ module GraphQL
 
       def self.emit_string(ts, te, meta, block:)
         quotes_length = block ? 3 : 1
-        ts += quotes_length
```

Comment:
This was changing `ts`, which seems to be used below for character positioning. In the character positioning case, I think we want it to be the original pointer?